roles/edpm_nova/tasks/configure.yml

```diff
@@ -118,3 +120,33 @@
     mode: '0600'
     owner: nova
     group: nova
+
+- name: get fact for tls_certs_enabled
+  set_fact: tls_certs_enabled="{{ lookup('env','tls_certs_enabled') }}"
+
+- name: copy cert file to {{edpm_nova_certs_dest}}
+  tags:
+    - configure
+    - nova
+  ansible.builtin.copy:
+    src: "{{ edpm_nova_certs_src }}/{{ inventory_hostname }}-tls.crt"
+    dest: "{{edpm_nova_certs_dest}}/tls.crt"
+  when: tls_certs_enabled
```

Comment:
so here we are using inventory_hostname to select the
relevant cert for this hsot and copy it to {{edpm_nova_certs_dest}}/tls.crt


so on each host the cert will be named 
tls.crt and it will be the cert for the current host +1